plugins/com.google.cloud.tools.eclipse.appengine.libraries.test/src/com/google/cloud/tools/eclipse/appengine/libraries/model/CloudLibrariesTest.java

```diff
@@ -38,13 +38,26 @@ public void testGetLibraries() {
   public void testGetClientApis() {
```

Comment:
Might this be better named `validateClientApis()`?

Reply:
It could be. I'm still in the habit of naming all test methods testFoo. Not sure I want to break that convention.